examples/simple_ffmpeg.rs

```diff
@@ -0,0 +1,78 @@
+use anyhow::Result;
+use compositor_pipeline::Pipeline;
+use serde_json::json;
+use signal_hook::{consts, iterator::Signals};
+use std::{env, fs, process::Command, sync::Arc, thread, time::Duration};
+use video_compositor::{http, state::State};
+
+use crate::common::write_example_sdp_file;
+
+#[path = "./common/common.rs"]
+mod common;
+
+fn main() {
+    ffmpeg_next::format::network::init();
+    let pipeline = Arc::new(Pipeline::new());
+    let state = Arc::new(State::new(pipeline));
+
+    thread::spawn(|| {
+        if let Err(err) = example() {
+            eprintln!("{err}")
+        }
+    });
+
+    http::Server::new(8001, state).start();
+
+    let mut signals = Signals::new([consts::SIGINT]).unwrap();
+    signals.forever().next();
+}
+
+fn example() -> Result<()> {
```

Comment:
Consider renaming this function to something more informative, like `spawn_input_and_output`